cloud/azure/deploy/api/httpproxy.go

```diff
@@ -80,7 +80,7 @@ func NewAzureHttpProxy(ctx *pulumi.Context, name string, args *AzureHttpProxyArg
 		}.ToUserIdentityPropertiesMapOutput()
 	}).(apimanagement.UserIdentityPropertiesMapOutput)
 
-	res.Service, err = apimanagement.NewApiManagementService(ctx, utils.ResourceName(ctx, name, utils.ApiManagementRT), &apimanagement.ApiManagementServiceArgs{
+	res.Service, err = apimanagement.NewApiManagementService(ctx, utils.ResourceName(ctx, args.App.Name+"-"+name, utils.ApiManagementRT), &apimanagement.ApiManagementServiceArgs{
```

Comment:
Prefixing the function name is to fix a duplicate URN error when deploying multiple http proxies with Azure